vehicles-utils/upload-renting-leasings/upload-renting-leasings.py

```
@@ -0,0 +1,453 @@
+        )
+
+        validate_total_calculations(
+            mapped_data["subtotalScheduledFee"],
+            mapped_data["scheduledFeeTaxType"],
+            mapped_data["scheduledFeeTax"],
+            mapped_data["scheduledFeeTotalAmount"],
+        )
+
+        return vehicle["id"], mapped_data
+    except Exception as e:
+        raise ValueError(f"Error al mapear fila: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)